stream_alert/rule_processor/metrics.py

```diff
@@ -0,0 +1,122 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import json
+
+from datetime import datetime
+
+import boto3
+
+from botocore.exceptions import ClientError
+
+from stream_alert.rule_processor import LOGGER
+
+
+class Metrics(object):
+    """Class to hold Rule Processor metric name and unit constants
+    This basically acts as an enum, allowing for the use of dot notation for
+    accessing properties and avoids doing dict lookups a ton.
+    """
```

Comment:
Nah, this is fine. I just saw `enum` in a comment and was making sure we knew about it.